.ddqa/config.toml

```
@@ -121,7 +121,7 @@ github_labels = ["team/remote-config"]
 jira_project = "FA"
 jira_issue_type = "QA"
 jira_statuses = ["To Do", "In Progress", "Done"]
-github_team = "fleet-automation"
+github_team = "Fleet"
```

Are you sure it is with an uppercase `F` ?
```suggestion
github_team = "fleet"
```

---

Is it case insensitive?

---

Sorry I thought it wasn't with the `F` in github. I approve 👍